exporters/elasticsearch/test/es_log_exporter_test.cc

```diff
@@ -14,6 +14,11 @@ namespace logs_api      = opentelemetry::logs;
 namespace nostd         = opentelemetry::nostd;
 namespace logs_exporter = opentelemetry::exporter::logs;
 
+TEST(ElasticsearchLogsExporterTests, Dummy)
+{
+  // to enable linking
```

Comment:
Is this empty test body expected?

Reply:
Yes, it seems at-least one TEST is required. I was getting unresolved symbols for gtest libary if there are no tests. We can remove this empty tests when the actual tests for EleasticSearch are working fine ( as of now they fail ).